test/common/network/utility_test.cc

```diff
@@ -150,8 +153,10 @@ TEST(NetworkUtility, ParseInternetAddress) {
   EXPECT_EQ(nullptr, Utility::parseInternetAddressNoThrow("0:0:0:0"));
   EXPECT_EQ(nullptr, Utility::parseInternetAddressNoThrow("fffff::"));
   EXPECT_EQ(nullptr, Utility::parseInternetAddressNoThrow("/foo"));
+#ifndef WIN32
   EXPECT_EQ(nullptr, Utility::parseInternetAddressNoThrow("[::]"));
   EXPECT_EQ(nullptr, Utility::parseInternetAddressNoThrow("[::1]:1"));
+#endif
```

Comment:
I think in the NoThrow case it makes sense to have the `#else` clause to show what happens on Windows.

Reply:
done.

Reply:
Same logic above at 142 (note an extra empty line there before #if), below at 237 & 251?

We might want to record a bug to track a different PR to the getaddrinfo call wherein our wrapper enforces a little more strictness, feel free to assign me to it.

Reply:
created the TODO with the bug number and added an extra space before the compiler conditions.